src/main/java/pages/JobPage.java

```diff
@@ -0,0 +1,88 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.util.List;
+import java.util.Random;
+
+public class JobPage extends BasePage{
+
+    private String jobCheckStr = "//*[text()='%s']/following-sibling::span";
+    private final By JobsLoc = By.xpath("//*[@class='job_list_company_title']");
+    private String checkLocStr = "//*[@id='jobsfilter-category']//*[text()='%s']/preceding-sibling::input";
+    private final WebDriverWait wait;
+    private final By randomJobLoc = By.xpath("//*[@class='job-inner job-item-title']");
+    private final By randomJobTitleLoc = By.xpath(".//*[@class='font_bold']");
+    private String jobTitleFirst;
+    private WebElement jobRandom;
+    private String url="/en/jobs";
+
+    public JobPage(WebDriver driver) {
+        super(driver);
+        wait = new WebDriverWait(driver, 20);
+    }
+
+    private WebElement CheckCount(String jobCount) {
+        String job = String.format(jobCheckStr, jobCount);
+        return driver.findElement(By.xpath(job));
+    }
+
+    public JobPage open(){
+        driver.get(BASE_URL+url);
+        return this;
+    }
+
+    private WebElement CheckButton(String jobCheck) {
```

Comment:
Please change method name so it will fit with the function that it actually does.